### PR TITLE
add woff2

### LINKED ISSUE #3586: Access to media.readthedocs.org/fonts/fontawesome-webfont.woff2 blocked by CORS policy
## Details

* Project URL: http://django-compressor.readthedocs.io/
* Build URL (if applicable):
* Read the Docs username (if applicable): 

## Expected Result

Page loaded without errors.

## Actual Result

> (index):1 Access to Font at 'https://media.readthedocs.org/fonts/fontawesome-webfont.woff2?v=4.6.3' from origin 'http://django-compressor.readthedocs.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://django-compressor.readthedocs.io' is therefore not allowed access. The response had HTTP status code 404.

### PR DESCRIPTION
**Pull request by fiennyangeln**
Added woff2

Closes #3586

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Version 2.1.6
 -------------
-
+ * `@fiennyangeln <http://github.com/fiennyangeln>`_: Add woff2 (`#3586 <https://github.com/rtfd/readthedocs.org/issues/3586>`_)
```
Comment (stsewd) [inline CHANGELOG.rst]: No need to edit the changelog here, this is done _automatically_ (or almost #3524)

Comment (fiennyangeln) [reply CHANGELOG.rst]: ok. I will change it

Comment (fiennyangeln) [reply CHANGELOG.rst]: Changed! thanks

**Comment by davidfischer** [comment]
This looks good.

What version of fontawesome is this? It doesn't seem to match the sha hash of the one from https://github.com/FortAwesome/Font-Awesome/releases/tag/v4.6.3.

**Comment by ericholscher** [comment]
I believe this actually got done in another PR. Thanks for this change though!

**Pull request closed without merging**